### ISSUE TITLE
[statement] State more clearly that Black Lives Matter

### ISSUE THREAD

**Issue by tobie** [w3c/idcg#25]
The phrase BLM appears once in the statement and once in the text:

> W3C Statement in support of Black Lives Matter

> As an organization, W3C believes that Black Lives Matter. We also stand in support of our Indigenous colleagues, colleagues of color, LGBTQI+ colleagues, and colleagues with disabilities.

The place in which it shows up in the text (near the bottom), the use of the terms "believes" and "in support of", and the apposition of a list of other under-presented and oppressed communities weakens the message.

Additionally, the terms "organization" and "colleagues" are confusing. Is this W3C staff talking about W3C colleagues, or is a statement from the broader membership?

My suggestion here would be to start the statement with something along the lines of: "Black Lives Matter" or "W3C wants to unequivocally state that Black Lives Matter" and remove the other sentence altogether.

**Comment by LJWatson** [comment]
We can't change the statement now because it's with the AC for review (where it remains member confidential for the time being).

**Comment by chaals** [comment]
I think we **could** change the statement, but the price of it is waiting another few weeks.

No statement will be perfect, and the timeliness is part of the effective statement itself. I don't think going another cycle of revisions for this would be any more helpful. 

So I agree that this issue might as well be closed without action.

**Comment by tobie** [comment]
I’m quite the confused. The WBS call for review asked for change suggestions and pointed to this repo to make those.

**Comment by LJWatson** [comment]
You're not the only one that's confused.

**Comment by tobie** [comment]
> I think the title of the survey "W3C Statement in Support of Black Lives Matter" is not part of the content that we are reviewing. @tobie does that make you more comfortable with the content?

Not really. I think the rest of my concerns still hold.

I do acknowledge and deeply empathize with the difficulty of releasing a statement that is acceptable to such a broad membership, btw. Thank you all for your work.

**Comment by torgo** [comment]
Tobie this statement needs to go out in a timely manner in order for it to be effective at all. I thin the question you need to ask yourself is "can I live with it?"

**Comment by tobie** [comment]
> Tobie this statement needs to go out in a timely manner in order for it to be effective at all.

I agree. I already think this is _really_ late, and I regret the additional August 18 delay which I assume is process-mandated.

> I thin the question you need to ask yourself is "can I live with it?"

I thought I had already addressed this by answering: "suggests changes, but supports adoption whether or not the changes are adopted" in the WBS CFC.

To be honest, I'm really confused by the process.

I was asked to provide my opinion using this channel, took time to do so, and feel like I've upset a bunch of unwritten rules in the process. I suggest wording the WBS CFC differently if what you want is just +1 or formal objection as answers.

For what it's worth, I feel like this statement is too little, too late. I would like W3C to have more agency as to what it can say publicly to avoid this in the future (and recognize that this is not the place to have a debate about this issue).

Feel free to ignore the issues I opened, I certainly don't want to delay this even further.

**Comment by koalie** [comment]
> I was asked to provide my opinion using this channel, took time to do so, and feel like I've upset a bunch of unwritten rules in the process. I suggest wording the WBS CFC differently if what is wanted is just +1 or formal objection as answers.

@tobie my apologies because I didn't think through all of the options we were going to offer in the form, and your feedback served as testing those.

**Comment by LJWatson** [comment]
Thanks @tobie.

The Process requires that the AC is given 4 weeks to review something it is asked to endorse. It was 6 weeks for most of the time of the pandemic, but was returned to 4 weeks in time for this review to go out.

The WBS form was left in its standard format for such reviews. The thinking was that it would be less confusing if it remained in the familiar format, but I think we can probably agree this didn't work out as planned, and @koalie  is already on the case. 

I appreciate that the statement might seem "too little too late", but the ID CG put a lot of time and effort into drafting the proposed statement and, more importantly, into getting some initial activities actually underway so that the statement could be backed up with concrete actions - in other words to protect against it being regarded as an empty statement.

The W3C has a set of [Policy Statement Guidelines (PSG)](https://www.w3.org/2014/01/PSGMatrix
) that are relevant to what W3C, specifically W3C team rather than the membership, can and cannot say on different types of policy. To raise an issue about this, head over to [ab-memberonly](https://github.com/w3c/ab-memberonly/issues/new).

Your comments are really welcome Tobie. Hope this explains some of the backdrop this is set against though.

**Comment by nigelmegitt** [comment]
I want to add my voice in support of @tobie here; reading this ticket it seems like a well thought through change was proposed, apparently completely within the stated terms of the review, and it was shot down as though it was obvious that no change could be accommodated. This kind of response doesn't feel right for us in this community. Good to see the speedy apologies.

If the AC review process does not allow acceptance contingent on a change, or acceptance but requesting a change, without sending the document back round a time-costly re-review loop, that doesn't seem right either. Asking AC to rubber-stamp a document with no prior notification that it was under preparation (and therefore no opportunity to contribute), and no opportunity to propose changes, under the duress of "we should have done this 4 weeks ago, so you'd better say yes", would reflect poorly on W3C. [My "would" there, instead of "does" is because I don't recall seeing a notification that this document was being prepared, but then a lot passes me by; apologies in advance if I missed that]

Either the marketing and comms staff need agency to respond in a timely way to current events, or we need a better way to collaborate speedily. The need to promote diversity and act against oppression is not a new thing, and W3C is not new to trying to address it: that agency should be derived from confidence that W3C is indeed representative of the diverse community it seeks to support. The "action" titled "Make W3C more welcoming" is very weak in this regard. Feeling welcome is good to have, but being actively included is the requirement.

**Comment by wareid** [comment]
I agree with everyone here that I wish we could have posted the statement earlier. 

I also think it should not have had to come from a member of a community group (here is where I take responsibility for this) I noticed W3C hadn't done anything specific, I went to the group I knew would share my concerns, we worked together to draft something that felt right for the tone of the organization and our goals of expressing support. 

Nigel and Tobie, your comments are important, but we're constrained by process here, and while I have a lot of thoughts on process (and I share Nigel's concerns about how it pertains here), we need to work within this system to get this action done. 

Should we open up issues within Process and W3M to address why we had to do this in the first place, YES. We can do that while we get this statement voted on. As for the statement itself, I hope we can all support it.

**Comment by LJWatson** [comment]
The ID CG was trying to move fast within a Process not designed for agility, and if we forgot to send a specific message to the AC about the draft statement, I'm sorry.

For reference, the AC was asked to [participate in ID CG](https://lists.w3.org/Archives/Member/w3c-ac-forum/2020AprJun/0118.html) as we started working on activities in response to a message from an [AC rep](https://lists.w3.org/Archives/Member/w3c-ac-forum/2020AprJun/0115.html), [and again](https://lists.w3.org/Archives/Member/w3c-ac-forum/2020AprJun/0149.html) as the ID CG began working on the proposed statement. It was [reiterated](https://lists.w3.org/Archives/Member/w3c-ac-forum/2020AprJun/0151.html) later in the same thread.

**Comment by jeffjaffe** [comment]
@tobie For clarification, the survey sent to the ac-forum allows ac reps to suggest changes, as is standard practice for all AC surveys.

**Comment by tobie** [comment]
Thanks for the additional clarifications and context, everyone, and for doing the work in the first place. Thanks @wareid for filing the related process issue.

**Comment by tobie** [comment]
As stated previously and indicated in my response to the CFC, I don't want to delay this statement further and can live with the current text. Feel free to ignore my suggestions if incorporating them would delay publication substantially or risk doing so.

**Comment by LJWatson** [comment]
@tobie we are now in the process of responding to all the comments we received during the AC review, thank you for yours.

>The place in which it shows up in the text (near the bottom), the use of the terms "believes" and "in support of", and the apposition of a list of other under-presented and oppressed communities weakens the message.

The statement will be published on its own page, the title of which will be something like "W3C supports Black Lives Matter", and the title will also be reflected in the primary heading of the page, giving the essential statement the highest prominence it can be given. Does this resolve your concern?

Removing the sentence from the statement itself, along with the rest of that paragraph, would be considered a substantive edit, meaning we would need to return the updated version to the AC for a further review. We're hesitant to do that because it has already been suggested that the timing of this statement could have been better.

It's also worth mentioning that the IDCG felt it was important to mention other under-represented and marginalised communities because we want to do better by all such groups.

>Additionally, the terms "organization" and "colleagues" are confusing. Is this W3C staff talking about W3C colleagues, or is a statement from the broader membership?

The statement is on behalf of W3C, including its team, members and participants. Colleagues is still used as a form of address by some within our community, though it is arguably an echo of a more formal mode of address.

We may be able to use some other words to resolve your concern, but reading through the [draft statement](https://github.com/w3c/idcg/wiki/Draft-BLM-statement) I have to admit I'm struggling to think of ways to do it without changing the essential meaning. If you have suggestions they would be gratefully received.

**Comment by tobie** [comment]
I'd like to defer to @kmcrayton7 advice, here.

**Comment by tobie** [comment]
Closing following decision made by the IDCG yesterday.
